src/db.rs

```diff
@@ -434,14 +548,14 @@ mod tests {
         let secid = Uuid::new_v4().as_hyphenated().to_string();
         let sec = Uuid::new_v4().as_hyphenated().to_string();
         let polid = 0;
-        insert_secret(&secid, &sec, Some(polid))?;
+        aw!(insert_secret(&secid, &sec, Option::<u64>::Some(polid)))?;
```

Comment:
[style] You can simplify this by helping the compiler with the type of `polid`:

```rust
let polid = 0u64;
aw!(insert_secret(&secid, &sec, Some(polid)))?;
```
(then the compiler can figure out which `Some` it actaully needs)

Reply:
Done.